src/Core/Logging/ILogServiceFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CodeMonkeys.Core.Logging
```

Comment:
why not in the `Interfaces` folder/namespace?
everything is (except your `Messaging` implementation, just realized that) is there...
we should be consistent

Reply:
As in the Messaging package I don't see any reason why we should have the Interfaces namespace at all. It doesn't bring any benefit at all. It should be clear that the Core only contains basic stuff like abstractions or maybe a enum and so on. And I like it more to have for example LogLevel in Core.Logging instead of just Core. It doesn't belong there

Reply:
Ok, can you please move the other interfaces then?

Reply:
Not in this PR